packages/@aws-cdk/aws-apprunner-alpha/lib/auto-scaling-configuration.ts

```diff
@@ -0,0 +1,198 @@
+   *
+   * Must be between 1 and 200.
+   *
+   * @default 100
+   */
+  readonly maxConcurrency?: number;
+
+  /**
+   * The maximum number of instances that a service scales up to.
+   * At most maxSize instances actively serve traffic for your service.
+   *
+   * Must be between 1 and 25.
+   *
+   * @default 25
+   */
+  readonly maxSize?: number;
+
+  /**
+   * The minimum number of instances that App Runner provisions for a service.
+   * The service always has at least minSize provisioned instances.
+   *
+   *
+   * Must be between 1 and 25.
+   *
+   * @default 1
+   */
+  readonly minSize?: number;
+}
+
+/**
+ * Attributes for the App Runner Auto Scaling Configuration.
+ */
+export interface AutoScalingConfigurationAttributes {
+  /**
+   * The name of the Auto Scaling Configuration.
+   */
+  readonly autoScalingConfigurationName: string;
+
+  /**
+   * The ARN of the Auto Scaling Configuration.
+   */
+  readonly autoScalingConfigurationArn: string;
+
+  /**
+   * The revision of the Auto Scaling Configuration.
+   */
+  readonly autoScalingConfigurationRevision: number;
+}
+
+/**
+ * Represents the App Runner Auto Scaling Configuration.
+ */
+export interface IAutoScalingConfiguration extends cdk.IResource {
+  /**
+   * The ARN of the Auto Scaling Configuration.
+   * @attribute
+   */
+  readonly autoScalingConfigurationArn: string;
+
+  /**
+   * The Name of the Auto Scaling Configuration.
+   * @attribute
+   */
+  readonly autoScalingConfigurationName: string;
+
+  /**
+   * The revision of the Auto Scaling Configuration.
+   * @attribute
+   */
+  readonly autoScalingConfigurationRevision: number;
+}
+
+/**
+ * The App Runner Auto Scaling Configuration.
+ *
+ * @resource AWS::AppRunner::AutoScalingConfiguration
+ */
+export class AutoScalingConfiguration extends cdk.Resource implements IAutoScalingConfiguration {
+  /**
+   * Imports an App Runner Auto Scaling Configuration from attributes
+   */
+  public static fromAutoScalingConfigurationAttributes(scope: Construct, id: string,
+    attrs: AutoScalingConfigurationAttributes): IAutoScalingConfiguration {
+    const autoScalingConfigurationArn = attrs.autoScalingConfigurationArn;
+    const autoScalingConfigurationName = attrs.autoScalingConfigurationName;
+    const autoScalingConfigurationRevision = attrs.autoScalingConfigurationRevision;
+
+    class Import extends cdk.Resource {
+      public readonly autoScalingConfigurationArn = autoScalingConfigurationArn
+      public readonly autoScalingConfigurationName = autoScalingConfigurationName
+      public readonly autoScalingConfigurationRevision = autoScalingConfigurationRevision
+    }
+
+    return new Import(scope, id);
+  }
+
+  /**
+   * Imports an App Runner Auto Scaling Configuration from its ARN
+   */
+  public static fromArn(scope: Construct, id: string, autoScalingConfigurationArn: string): IAutoScalingConfiguration {
```

Comment:
I guess `fromArn` won't pass the linter but I am not 100% sure. I guess `fromAutoScalingConfigurationArn` would be expected. But I might be wrong.